packages/strapi-admin/admin/src/components/EventInput/tests/formatValue.test.js

```diff
@@ -0,0 +1,15 @@
+import formatValue from '../utils/formatValue';
+
+describe('utils | formatValue', () => {
+  it('should format array to object', () => {
+    const initialValue = ['entry.create', 'entry.update', 'media.delete'];
+    const expectedValue = {
+      entry: ['entry.create', 'entry.update'],
+      media: ['media.delete'],
+    };
+
+    const formattedValue = formatValue(initialValue);
+
+    expect(expectedValue).toEqual(formattedValue);
```

Comment:
Usually we write:

`expect(formatValue(initialValue)).toEqual(expectedValue);`